tensorboard/webapp/tbdev_upload/tbdev_upload_button_component.ts

```diff
@@ -0,0 +1,50 @@
+/* Copyright 2020 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+import {Component} from '@angular/core';
+import {MatDialog} from '@angular/material/dialog';
+
+import {TbdevUploadDialogComponent} from './tbdev_upload_dialog_component';
+
+@Component({
+  selector: 'tbdev-upload-button',
+  template: `
+    <button mat-raised-button color="primary" (click)="openDialog()">
```

Comment:
does it need to be a raised button. It is a bit ugly....

Reply:
Agree it should not be a raised button. I have to follow up with Abdalla about it so will tackle this in a subsequent CL.